easypost/__init__.py

```diff
@@ -291,21 +294,26 @@ def request_raw(self, method, url, params=None, apiKeyRequired=True):
 
         if hasattr(ssl, 'OPENSSL_VERSION'):
             ua['openssl_version'] = ssl.OPENSSL_VERSION
-
-        headers = {
-            'X-Client-User-Agent': json.dumps(ua),
-            'User-Agent': USER_AGENT,
-            'Authorization': 'Bearer %s' % my_api_key,
-            'Content-type': 'application/x-www-form-urlencoded'
-        }
-
+        if isRating:
+            headers = {
+                'X-Client-User-Agent': json.dumps(ua),
+                'User-Agent': USER_AGENT,
+                'Authorization': 'Bearer %s' % my_api_key,
+                'Content-type': 'application/json' 
+            }
+        else:
+            headers = {
+                'X-Client-User-Agent': json.dumps(ua),
+                'User-Agent': USER_AGENT,
+                'Authorization': 'Bearer %s' % my_api_key,
+                'Content-type': 'application/x-www-form-urlencoded' 
```

Comment:
Why was "type" (Content-type) not capitalized?
Can you change it and test that it still works correctly?